config/sync/core.entity_form_display.media.external_video.default.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   config:
     - field.field.media.external_video.field_conference
     - field.field.media.external_video.field_description
+    - field.field.media.external_video.field_event_
```

Comment:
Keep an eye on machine names when they're generated - `field_event` is a better machine name than `field_event_`

Reply:
Thank u. I'll keep in mind next time.